README.md

```diff
@@ -219,11 +219,35 @@ the following contents:
 URL_ROOT = 'http://localhost:8000/a/{{DOMAIN}}'
 ```
 
+##### Running Formplayer in Docker
+
+Please refer to FormPlayer's install instructions under "[Running in Docker](https://github.com/dimagi/formplayer#running-in-docker)".
+
 If you are on Mac, don't bother trying to run this in Docker. There seems to be some kind of bug.
-Check out the install instructions under "[Building and Running]
-(https://github.com/dimagi/formplayer#building-and-running)".
+Instead, try running formplayer from a .jar file
+
+##### Running formplayer.jar
```

Comment:
@jmtroth0 @pr33thi @gbova If any of you don't yet have formplayer working, would you mind trying out this process and letting me know if it worked for you, and whether I missed any steps?